gensim/models/phrases.py

```diff
@@ -209,12 +209,12 @@ def load(cls, *args, **kwargs):
         model = super(PhrasesTransformation, cls).load(*args, **kwargs)
         # update older models
         # if value in phrasegrams dict is a tuple, load only the scores.
-        try:
-            for components, scores in model.__dict__['phrasegrams'].items():
-                if isinstance(scores, tuple):
-                    model.__dict__['phrasegrams'][components] = scores[1]
-        except KeyError:
-            pass
+        if model.phrasegrams:
+            components = model.phrasegrams.keys()
+            for component in components:
+                score = model.phrasegrams[component]
+                if isinstance(score, tuple):
+                    model.phrasegrams[component] = score[1]
```

Comment:
Deserves a comment: what is `score[1]`?

Or even better, unroll the tuple into properly named variables (`x, y, z = score`) and then assign that.